test/module/irohad/ametsuchi/ametsuchi_fixture.hpp

```diff
@@ -97,8 +99,14 @@ namespace iroha {
 
       std::shared_ptr<StorageImpl> storage;
 
+      // generate random valid dbname
+      std::string dbname_ = "d"
+          + boost::uuids::to_string(boost::uuids::random_generator()())
+                .substr(0, 8);
```

Comment:
why?

Reply:
sorry, i mean the size

Reply:
first 8 characters of uuid are letters. 9th -- dash